lib/parity/environment.rb

```diff
@@ -52,11 +52,24 @@ def restore
         Backup.new(
           from: arguments.first,
           to: environment,
-          additional_args: arguments.drop(1).join(" ")
+          additional_args: additional_restore_arguments
         ).restore
       end
     end
 
+    def additional_restore_arguments
+      (arguments.drop(1) + [restore_confirmation_argument]).
+        compact.
+        join(" ")
+    end
+
+    def restore_confirmation_argument
+      if !["production", "development"].include?(environment) &&
+        Parity.config.bypass_confirm_overwrite_for_remote_databases
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.